tests/generic_config_updater/test_dynamic_acl.py

```diff
@@ -1154,6 +1226,12 @@ def test_gcu_acl_forward_rule_replacement(rand_selected_dut, ptfadapter, setup,
     dynamic_acl_create_forward_rules(rand_selected_dut)
     dynamic_acl_create_secondary_drop_rule(rand_selected_dut, setup)
     dynamic_acl_replace_rules(rand_selected_dut)
+
+    if setup["is_dualtor_aa"]:
```

Comment:
I suggest handling the logic for `dualtor-aa` inside the helper function so that the test code is much cleaner.